staging/src/k8s.io/cri-api/pkg/apis/testing/utils.go

```diff
@@ -16,23 +16,6 @@ limitations under the License.
 
 package testing
 
-import (
-	"fmt"
-
-	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
-)
-
-// BuildContainerName creates a unique container name string.
-func BuildContainerName(metadata *runtimeapi.ContainerMetadata, sandboxID string) string {
```

Comment:
These are genuinely useful.

Reply:
I've added it back, and the comment above, PTAL @thockin